homeassistant/components/automation/__init__.py

```diff
@@ -330,8 +318,8 @@ def extra_state_attributes(self):
         }
         if self.action_script.supports_max:
             attrs[ATTR_MAX] = self.action_script.max_runs
-        if self._id is not None:
-            attrs[CONF_ID] = self._id
+        if self._attr_unique_id is not None:
+            attrs[CONF_ID] = self._attr_unique_id
```

Comment:
The pattern of accessing the attribute directly instead of the property is used in some other places as well.
Here's a PR documenting that we want the property to be accesses: https://github.com/home-assistant/developers.home-assistant/pull/999/files